test/test_merge_db.py

```diff
@@ -156,3 +157,33 @@ def test_empty(self):
         result2 = db.combine(*emptyWindows)
 
         self.assertEqual(result2, db.Window.make_empty(w_si))
+
+    def test_window_timestamp_attribute(self):
+        w_si = si.Window(3, 7, 250)
+
+        time_a = 3
+
+        data_a = np.arange(1, 22).reshape((7, 3))
+
+        A = db.Window(w_si, time_a, data_a)
+        B = copy.deepcopy(A)
+
+        self.assertEqual(A, B)
+
+    def test_window_construct_from_other(self):
+        time = 3
+        channels = 2
+        samples = 20
+        sampling_rate = 250
+
+        w_si = si.Window(channels, samples, sampling_rate)
+
+        time_a = 3
+        data = np.arange(1, 41, dtype=np.float64).reshape(samples, channels)
+        window = db.Window(w_si, time, data)
+
+        try:
+            new_window = db.Window(w_si, time, [window])
```

Comment:
This is incorrect, you should not accept array of db.Window as an argument, that does not make any sense.
In principle it could be one of:
* db.Window
* array of db.SingleWindow
* array of np.ndarray 
* np.ndarray with 3 dimensions
I don't say you need implement all of them right now, you can choose one which is required for you right now